packages/vscode-dependencies-validation/package.json

```diff
@@ -18,10 +18,31 @@
     "directory": "packages/npm-dependencies-validation"
   },
   "main": "./dist/src/extension",
+  "activationEvents": [
+    "*"
+  ],
+  "contributes": {
+    "configuration": {
+      "title": "NPM Dependencies Validation",
+      "properties": {
+        "dependenciesValidation.enableAutoFix": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enables automatic dependencies misalignment fixing"
+        },
+        "dependenciesValidation.delayAutoFix": {
+          "type": "number",
+          "default": 0,
+          "description": "Delays automatic dependencies misalignment fixing on startup (seconds)"
+        }
+      }
+    }
+  },
   "scripts": {
     "ci": "npm-run-all clean compile bundle package",
     "clean": "rimraf ./dist *.vsix ./coverage ./.nyc_output",
     "compile": "tsc -p ./",
+    "watch": "tsc -p ./ --watch",
```

Comment:
I asked somewhere else, but maybe have not noticed the reply there.
do we need a compile watch script here?
considering we have a root level watch for the whole mono-repo?